libraries/node-core-library/src/Resolve.ts

```diff
@@ -3,15 +3,23 @@
 
 import { sync } from 'resolve';
 import * as path from 'path';
-import { FileSystem, PackageJsonLookup } from '@rushstack/node-core-library';
+
+import { PackageJsonLookup } from './PackageJsonLookup';
+import { FileSystem } from './FileSystem';
 
 const packageJsonLookup: PackageJsonLookup = new PackageJsonLookup();
 
+/**
+ * @alpha
+ */
 export interface IResolvePackageOptions {
   doNotResolveSymlinks: boolean;
```

Comment:
Why do we need this? When would it ever be useful?

Reply:
Removed.